.yamato/all-hdrp.yml

```diff
@@ -20,8 +20,26 @@ PR_HDRP_trunk:
     dependencies:
       -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_Standalone_mono_Linear_trunk
          rerun: on_new_revision
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_mono_Linear_trunk
+         rerun: on_new_revision
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_XR_mono_Linear_trunk
+         rerun: on_new_revision
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_editmode_mono_Linear_trunk
+         rerun: on_new_revision
+      -  path: .yamato/hdrp-win-dx12.yml#HDRP_Win_DX12_playmode_mono_Linear_trunk
+         rerun: on_new_revision
+      -  path: .yamato/hdrp-win-vulkan.yml#HDRP_Win_Vulkan_playmode_mono_Linear_trunk
+         rerun: on_new_revision
+      -  path: .yamato/hdrp-osx-metal.yml#HDRP_OSX_Metal_playmode_mono_Linear_trunk
+         rerun: on_new_revision
+      -  path: .yamato/hdrp-linux-vulkan.yml#HDRP_Linux_Vulkan_playmode_mono_Linear_trunk
+         rerun: on_new_revision
+      -  path: .yamato/hdrp-linux-vulkan.yml#HDRP_Linux_Vulkan_Standalone_mono_Linear_trunk
+         rerun: on_new_revision
       -  path: .yamato/all-hdrp_dxr.yml#PR_HDRP_DXR_trunk
          rerun: on_new_revision
+      -  path: .yamato/all-hdrp_hybrid.yml#PR_HDRP_Hybrid_trunk
```

Comment:
Should Hybrids be excluded still?

Reply:
Good question. I tried finding where the Nightly Hybrid was called back in April https://github.com/Unity-Technologies/Graphics/tree/4a23bfca06c6b533131042e410ea0fef0ba06503/.yamato, but I can't find any pipeline calling it, or trigger.
Aren't we supposed to run some hybrid tests, at least in the nightly?